app/controllers/glossaries_controller.rb

```diff
@@ -33,7 +33,7 @@ def edit
 
 
   def update_params
-    params.require(:glossary).permit(:name, :json, :user_id, :legacy_glossary_resource_id, :project_id, :project)
+    params.fetch(:glossary, {}).permit!
```

Comment:
We noticed that in glossaries_controller_spec (not the API version) we test for the the glossaries controller to be able to take in any params, and we had to make this change in order to support that. Not sure if this even matters if we aren't using the non-API version anymore.

Reply:
We should probably land on the side of safety and be explicit on the parameters.  That might mean updating the spec test.